### PR TITLE
Oolite 1.80 added

### PR DESCRIPTION
**Pull request by growler**

### CODE REVIEW

File: Casks/oolite.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask :v1 => 'oolite' do
+  version '1.80'
+  sha256 'da6dc76f526dccf29c7c2180353f88979778dd26630cb8c032eaa437c6d197f2'
+
+  url 'https://github.com/OoliteProject/oolite/releases/download/1.80b/Oolite-1.80.zip'
```
Comment (vitorgalvao) [inline Casks/oolite.rb]: Needs a `# github.com is the official download host per the vendor homepage` comment above this line ([example](https://github.com/caskroom/homebrew-cask/blob/cb487ad405f74cb1f26b93e3f3e8412c82a22e34/Casks/0ad.rb#L5)).

Comment (vitorgalvao) [inline Casks/oolite.rb]: As long as you’ll be making changes, you can change this `url "https://github.com/OoliteProject/oolite/releases/download/#{version}b/Oolite-#{version}.zip"`.

Comment (growler) [inline Casks/oolite.rb]: Thank you. Sorry for the mess.

**Comment by vitorgalvao** [comment]
Thank you, could you please [squash your commits](http://davidwalsh.name/squash-commits-git)?

**Comment by growler** [comment]
Next time for sure. Thank you for pointing that out.

**Comment by vitorgalvao** [comment]
Something went wrong. Do you need help? I can do it for you, this time, still preserving your contribution.

**Comment by growler** [comment]
Yep, I messed things up. I guess the best way would be to close this request and open a new one.

**Pull request closed without merging**